api/src/main/scala/com/gu/adapters/http/Authentication.scala

```diff
@@ -79,29 +176,41 @@ object AuthenticationService extends LazyLogging {
     // ExecutorService returned is a ThreadPoolExecutor.
     // Explicitly cast to this type so that thread pool can be monitored
     // (e.g. get access to active thread count etc).
-    val threadPool = Executors.newFixedThreadPool(blockingThreads).asInstanceOf[ThreadPoolExecutor]
+    val threadPool = Executors
+      .newFixedThreadPool(blockingThreads)
+      .asInstanceOf[ThreadPoolExecutor]
     AuthenticationServiceThreadPoolMonitorer.monitorThreadPool(threadPool)
 
     // Access token that's 'safe' to log e.g secret_token -> sec**********
     // Assumes size of  token significantly greater than size 3.
-    val scrubbedAccessToken = config.accessToken.zipWithIndex
-      .map { case (c, i) => if (i < 3) c else '*' }
-      .mkString
+    val scrubbedAccessToken = config.accessToken.zipWithIndex.map {
+      case (c, i) => if (i < 3) c else '*'
+    }.mkString
 
     val uri = Uri.unsafeFromString(config.apiUrl)
 
     // Log parameters to be sure they are correct.
-    logger.info(s"initialising identity auth service - url: $uri, access token: $scrubbedAccessToken")
+    logger.info(
+      s"initialising identity auth service - url: $uri, access token: $scrubbedAccessToken"
```

Comment:
There doesn't seem to be much value in logging the scrubbed access token here.  I suppose we just need to know that there was an access token in the request

Reply:
Yeah this already existed, just changed thanks to formatting, I'm not going to change around existing functionality.